streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetricsImpl.java

```diff
@@ -117,7 +118,7 @@ public Sensor addLatencyAndThroughputSensor(String scopeName, String entityName,
 
         // first add the global operation metrics if not yet, with the global tags only
         Sensor parent = metrics.sensor(sensorName(operationName, null), recordingLevel);
-        addLatencyMetrics(scopeName, parent, "all", operationName, tagMap);
+        addLatencyMetrics(scopeName, parent, null, operationName, tagMap("all", "all"));
```

Comment:
Another issue is that, for state-store-metrics, since the per store metrics do not have tags, doing this will result in a bit mis-match hierarchies like this:

![screen shot 2017-07-09 at 7 14 54 pm](https://user-images.githubusercontent.com/10729597/28000551-37d279f8-64db-11e7-8b27-4654823e12c9.png)

Note the "all" attributes is grouped with the other stores at the same level; instead what we want is to have "store-name" in the tags as well so that under `stream-[store-type]-state-metrics` we will have a first layer of "storeName1", "storeName2" .. and "all" (whose tags would be "store-name", "all").